src/thing_list.c

```diff
@@ -3514,7 +3514,48 @@ struct Thing *get_creature_of_model_training_at_subtile_and_owned_by(MapSubtlCoo
 
 struct Thing *get_nearest_object_at_position(MapSubtlCoord stl_x, MapSubtlCoord stl_y)
 {
-  return _DK_get_nearest_object_at_position(stl_x, stl_y);
+  // return _DK_get_nearest_object_at_position(stl_x, stl_y);
+  long OldDistance = 0x7FFFFFFF;
```

Comment:
there is a constant LONG_MAX or something like that

Reply:
Where's LONG_MAX defined? I can't find it anywhere.

Reply:
It seems to be some sort of C specific thing. LONG_MAX is used a lot in the KeeperFX code, just search the code for examples.